### DISCUSSION TITLE
1Panel应用商店是如何检测应用更新的?

### DISCUSSION THREAD

**Discussion by QingFeng-awa** [1Panel-dev/1Panel#6727] [General]
一直以来都挺疑惑1Panel是怎么检测应用更新的
每次软件更新1Panel应用商店都有延迟,更新应用列表也没用。
比如说飞致云的另一个项目Halo,2.20.0LTS发布了快1天了,到现在应用商店也没提示更新。
你们总不能是手动编辑的应用列表的吧……

**Comment by sucooer** [comment]
同问，1panel市场最新版才2.19.3，早知道用docker compose安装了

**Comment by wanghe-fit2cloud** [comment]
目前 Halo 已经发布了 v2.20.3 版本，等稳定后会推送更新。

**Reply by wanghe-fit2cloud** [reply]
<img width="387" alt="image" src="https://github.com/user-attachments/assets/189c4c1b-a2aa-48cf-a611-3a214ab215f6">
